app/services/github_handler.rb

```diff
@@ -45,6 +45,12 @@ def handle_review_request
     pr.review_requests.create!(data: payload, owner: owner, reviewer: reviewer)
   end
 
+  def handle_review_removal
+    reviewer = User.find_by(github_id: payload['requested_reviewer']['id'])
+    pr = PullRequest.find_by(github_id: payload['pull_request']['id'])
+    pr.review_requests.find_by!(reviewer: reviewer).destroy!
```

Comment:
do we care about an exception if the review_request isn't found?

maybe we can do `pr.review_requests.find_by(reviewer: reviewer)&.destroy!`

thougths?

Reply:
Do we want to destroy these requests? Or mark them as invalid?

For example, if person A sends a review request to person B and then removes it after 3 days, do we want that review request to affect the statistics?

I think so but what does everyone think?

@anthfb @matiasmansilla1989 @santib